### ISSUE TITLE
[BUG] No micrographs found at location .../10064/data/*.mrc/*.mrc

### ISSUE THREAD

**Issue by wlugmayr** [KudryashevLab/TomoBEAR#16]
**Describe the bug**

When using the interactive Matlab TomoBear version the file extension is duplicated. This worked in older TomoBEAR versions and works with the compiled MATLAb standalone executable.

**Project JSON file**
ribosome_empiar_10064_dynamo.json

```JSON
        "data_path": "/gpfs/cssb/software/tutorials/empiar_data/EMPIAR-10064-CTEM/10064/data/*.mrc",
```

**Error message**
Error using getOriginalMRCsorTIFs
ERROR: No micrographs found at location /gpfs/cssb/software/tutorials/empiar_data/EMPIAR-10064-CTEM/10064/data/*.mrc/*.mrc

Error in getFileCountChanged (line 2)
[file_list, ~] = getOriginalMRCsorTIFs(configuration);

Error in LocalPipeline/execute (line 465)
                                file_count_changed = getFileCountChanged(configuration.general);

Error in runPipeline (line 131)
            pipeline.execute(starting_tomogram, ending_tomogram, step, gpu);

Error in runTomoBear (line 15)
runPipeline(compute_environment, configuration_path, default_configuration_path, starting_tomogram, ending_tomogram, step, gpu);

Error in run_ribosome (line 1)
runTomoBear("local", "./ribosome_empiar_10064_dynamo.json", "/gpfs/cssb/software/centos7/x86_64/tomobear/0.2.0/configurations/defaults.json")

**To Reproduce**
Run the tutorial in the MATLAB GUI (matlab/R2022b).

**Additional context**
When using no extensions it gets the error:

INFO:DATA_PATH: /gpfs/cssb/software/tutorials/empiar_data/EMPIAR-10064-CTEM/10064/data/
Error using getUnprocessedTomograms
ERROR: Multiple unknown extensions found!

Error in LocalPipeline/execute (line 178)
                    [dynamic_configuration, file_count_changed] = getUnprocessedTomograms(configuration.general, log_file_id);

Error in runPipeline (line 131)
            pipeline.execute(starting_tomogram, ending_tomogram, step, gpu);

Error in runTomoBear (line 15)
runPipeline(compute_environment, configuration_path, default_configuration_path, starting_tomogram, ending_tomogram, step, gpu);

Error in run_ribosome (line 1)
runTomoBear("local", "./ribosome_empiar_10064_dynamo.json", "/gpfs/cssb/software/centos7/x86_64/tomobear/0.2.0/configurations/defaults.json")

**Comment by ArtsemiY** [comment]
Hi @wlugmayr !
I do apologize that it took much more than expected, but the newly published [release v0.3.0](https://github.com/KudryashevLab/TomoBEAR/releases/tag/v0.3.0) (w/o binary this time) should now solve this issue in the full MATLAB version of the TomoBEAR.

**Comment by ArtsemiY** [comment]
I assume that this issue was resolved and could be closed, otherwise you are welcome to re-open it.
